CHANGELOG.next.asciidoc

```diff
@@ -503,7 +503,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Release aws ebs metricset as GA. {pull}14312[14312] {issue}14060[14060]
 - Add Kafka JMX metricsets. {pull}14330[14330]
 - Add metrics to envoyproxy server metricset and support for envoy proxy 1.12. {pull}14416[14416] {issue}13642[13642]
-- Add module for ActiveMQ. {pull}14788[14788]
```

Comment:
Hmmm in backport, the PR number should always keep the same as the original PR I think. So both of these should be 14840.

Reply:
You're right. Fixed!